coalahtml/_coalahtml/bower.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "coala-html",
   "dependencies": {
-    "angular": "^1.4.0",
-    "angular-route": "^1.4.0",
-    "bootstrap": "^3.3.6",
-    "google-code-prettify": "^1.0.4",
-    "angular-sanitize": "^1.5.0",
-    "bootstrap-material-design": "^0.5.10"
+    "angular": "^1.6.2",
+    "angular-route": "^1.6.2",
+    "bootstrap": "^3.3.7",
+    "google-code-prettify": "^1.0.5",
+    "angular-sanitize": "^1.6.2",
+    "bootstrap-material-design": "^4.0.2"
```

Comment:
@jayvdb I have another issue assigned to myself, which is adding pagination to the main errors page in `coala-html`. To include that, I was planning to use [`ui-bootstrap`](https://angular-ui.github.io/bootstrap/), however, the `angular version 1.4.0` is the oldest version which the tool supports, and hence decided to upgrade. For this, I used [`npm-check-updates`](https://www.npmjs.com/package/npm-check-updates). You are right, there is actually no need to upgrade bootstrap and bootstrap MD. What all should I upgrade then?